src/null-grammar.js

```diff
@@ -0,0 +1,13 @@
+/** @babel */
+
+import {Disposable} from 'event-kit'
+
+module.exports = Object.freeze({
```

Comment:
Since we're using the `import` notation above, what do you think about changing this into `export default Object.freeze(…)`?